### PR TITLE
Featured chapters methodology japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

Please review.

### CODE REVIEW

File: src/templates/ja/2019/methodology.html [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+          <tr>
+            <td>Download throughput</td>
+            <td>1.475 Mbps</td>
+          </tr>
+          <tr>
+            <td>Upload throughput</td>
+            <td>0.675 Mbps</td>
+          </tr>
+          <tr>
+            <td>RTT</td>
```
Comment (bazzadp) [inline src/templates/ja/2019/methodology.html:285]: Should we translate all these config titles (or at least the first 3)?

File: src/templates/ja/2019/methodology.html [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+      <p>
+        <a href="https://www.wappalyzer.com/">Wappalyzer</a>は、ウェブページで使用されている技術を検出するためのツールです。<a href="https://www.wappalyzer.com/technologies">65のカテゴリ</a>があり、JavaScriptフレームワークからCMSプラットフォーム、さらには暗号通貨の採掘者に至るまで、テストされた技術の範囲があります。1,200以上の技術がサポートされています。
+      </p>
+
+      <p>
+        HTTP Archiveは、すべてのウェブページに対して最新バージョンのWappalyzerを実行します。2019年7月現在、Web AlmanacはWappalyzerの<a href="https://github.com/AliasIO/Wappalyzer/releases/tag/v5.8.3">5.8.3バージョン</a>を使用しています。
+      </p>
+
+      <p>
+        Wappalyzerは、WordPress、Bootstrap、jQueryのような開発者ツールの人気を分析する多くの章を強力にしています。例えば、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce')}}">Eコマース</a> と <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms')}}">CMS</a> の各章は、それぞれの <a href="https://www. wappalyzer.com/categories/ecommerce">Eコマース</a> と <a href="https://www.wappalyzer.com/categories/cms">CMS</a> の各チャプターは、Wappalyzer が検出した技術のカテゴリに大きく依存しています。
```
Comment (bazzadp) [inline src/templates/ja/2019/methodology.html:313]: Extra space breaking the link:
```suggestion
        Wappalyzerは、WordPress、Bootstrap、jQueryのような開発者ツールの人気を分析する多くの章を強力にしています。例えば、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce')}}">Eコマース</a> と <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms')}}">CMS</a> の各章は、それぞれの <a href="https://www.wappalyzer.com/categories/ecommerce">Eコマース</a> と <a href="https://www.wappalyzer.com/categories/cms">CMS</a> の各チャプターは、Wappalyzer が検出した技術のカテゴリに大きく依存しています。
```

File: src/templates/ja/2019/methodology.html [hunk 3]
```diff
@@ -0,0 +1,433 @@
+{% extends "base/2019/methodology.html" %}
+
+{% block title %}方法論 | The Web Almanac by HTTP Archive{% endblock %}
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:3]: I missed, translated.

File: src/templates/ja/2019/methodology.html [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+      <p>
+        データセットの詳細については、<a href="https://web.dev/chrome-ux-report-bigquery">BigQueryでChrome UXレポートを使用する</a>のガイド<a href="https://web.dev/">web.dev</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="third-party-web"><a href="#third-party-web">サードパーティウェブ</a></h3>
+
+      <p>
+        <a href="https://www.thirdpartyweb. today/">サードパーティウェブ</a>は、<a href="{{ url_for(' contributors', year=year, lang=lang, _anchor='patrickhulce')}}">Patrick Hulce</a>の研究プロジェクトで、<a href="{{ url_for('chapter'. year=year, lang=lang, chapter='サードパーティ')}}">サードパーティ</a>の章では、HTTP ArchiveとLighthouseのデータを使用して、サードパーティのリソースがウェブに与える影響を特定して分析しています。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:341]: Sorry, I fixed.

File: src/templates/ja/2019/methodology.html [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+{% block index %}
+      <ul>
+        <li><a href="#overview">概要</a></li>
+        <li><a href="#dataset">データセットについて</a>
+          <ul>
+            <li><a href="#websites">ウェブサイト</a></li>
+            <li><a href="#metrics">メトリクス</a></li>
+          </ul>
+        </li>
+        <li><a href="#tools">Tools</a>
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:23]: Thanks, I fixed.

File: src/templates/ja/2019/methodology.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:285]: At first, I don't need translate.
But Now, I thought need translate at first 3.
I translated.

File: src/templates/ja/2019/methodology.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:313]: Sorry, I missed.
I fixed.

File: src/templates/ja/2019/methodology.html [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+        Web Almanacの発端は、2019年1月に<a href="https://discuss.httparchive.org/t/planning-the-web-almanac-2019/1553">HTTP Archiveフォーラムへの投稿</a>でイニシアチブを説明し、支援を集めることから始まりました。2019年3月には、Webコミュニティの誰もがチャプターやメトリクスのアイデアを書き込むことができる<a href="http://bit.ly/web-almanac-brainstorm">公開ブレインストーミング文書</a>を作成しました。これは、私たちがコミュニティにとって重要なことに焦点を当て、プロセスに含まれる多様な声のセットを持っていることを確認するための重要なステップでした。
+      </p>
+
+      <p>
+        ブレーンストーミングの結果、20の章が固まり、私たちは<a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/2">各章に主題専門家とレビュワーを割り当てる作業を開始しました</a>。このプロセスには、この規模のプロジェクトにボランティアを参加させるという課題があったため、いくつかの固有の偏りがありました。したがって、寄稿者の多くは同じ専門家サークルのメンバーです。Web Almanacの今後のエディションの明確な目標の1つは、著者やレビュワーとしての、代表性の低い、異質な声をより多く取り入れることを奨励することです。
+      </p>
+
+      <p>
+        私たちは2019年5月から6月にかけて、人々と支部とのペアリングを行い、各支部を構成する個々の指標を最終的に決定するために彼らの意見を聞きました。
+        We spent May through June 2019 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html:386]: Yes, I forgot delete.
I deleted this line.

File: src/templates/ja/2019/methodology.html [hunk 7]
```diff
@@ -0,0 +1,432 @@
...
+      <p>
+        データセットの詳細については、<a href="https://web.dev/chrome-ux-report-bigquery">BigQueryでChrome UXレポートを使用する</a>のガイド<a href="https://web.dev/">web.dev</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="third-party-web"><a href="#third-party-web">サードパーティウェブ</a></h3>
+
+      <p>
+        <a href="https://www.thirdpartyweb.today/">サードパーティウェブ</a>は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='patrickhulce')}}">Patrick Hulce</a>の研究プロジェクトで、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='サードパーティ')}}">サードパーティ</a>の章では、HTTP ArchiveとLighthouseのデータを使用して、サードパーティのリソースがウェブに与える影響を特定して分析しています。
```
Comment (bazzadp) [inline src/templates/ja/2019/methodology.html:341]: Spotted one more error:
```suggestion
        <a href="https://www.thirdpartyweb.today/">サードパーティウェブ</a>は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='patrickhulce')}}">Patrick Hulce</a>の研究プロジェクトで、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='third-parties')}}">サードパーティ</a>の章では、HTTP ArchiveとLighthouseのデータを使用して、サードパーティのリソースがウェブに与える影響を特定して分析しています。
```

Comment (ksakae1216) [reply src/templates/ja/2019/methodology.html]: Sorry...

I fixed.

**Pull request merged**